packages/extension-ui/src/hooks/useLedger.ts

```diff
@@ -102,7 +106,7 @@ export default function useLedger (genesis?: string | null, accountIndex = 0, ad
     setError(null);
     setWarning(null);
 
-    ledger.getAddress(false, accountIndex, addressOffset)
+    ledger.getAddress(chainInfo?.ss58Format || 0, false, accountIndex, addressOffset)
```

Comment:
Maybe we can do an `assert` here - We just want to make sure the ss58 is always for certain what is present in the ss58 format and can't resolve to `0`